word_guess.rb

```diff
@@ -0,0 +1,163 @@
+				puts %! #{@move[@wrong_count]}   |   | #{@dots[@wrong_count].colorize(:green)} #{@p3} !.colorize(:color => :green, :background => :black).blink
+				puts "  #{@move[@wrong_count]}  '^^^' #{@spaces[@wrong_count]}  #{@p4}  ".colorize(:color => :green, :background => :black).blink
+				puts "===================================".colorize(:color => :green, :background => :black).blink
+				puts "                                   ".colorize(:color => :green, :background => :black).blink
+				puts "Word: #{@game_word_array.join(" ")}   #{@space1}".colorize(:color => :green, :background => :black)
+				puts "letters guessed: #{@guessed_letters.join(" ")} #{@space2}".colorize(:color => :green, :background => :black)
+				puts "Word is #{@secret_word} #{@space3}".colorize(:color => :green, :background => :black)
+				puts "Number of guesses left: #{5-@wrong_count}          ".colorize(:color => :green, :background => :black) 
+				puts "===================================".colorize(:color => :green, :background => :black).blink 
+				puts "CONGRATS!!! YOU WON!!!!!!".colorize(:color => :green, :background => :black).blink
+				puts
+				exit
+			end
+
+		elsif @wrong_count == 5
+			@space1 = " "*(27 - (@game_word_array.length*2))
+			@space2 = " "*(19 - (@guessed_letters.length*2))
+			@space3 = " "*(26 - (@game_word_array.length))
+			puts "===================================".colorize(:color => :red, :background => :black).blink
+			puts "   #{@move[@wrong_count]}  .-.  #{@spaces[@wrong_count]}  #{@p1}  ".colorize(:color => :red, :background => :black).blink
+			puts "   #{@move[@wrong_count]} | OO| #{@spaces[@wrong_count]} #{@p2} ".colorize(:color => :red, :background => :black).blink
+			puts %! #{@move[@wrong_count]}   |   | #{@dots[@wrong_count].colorize(:green)} #{@p3} !.colorize(:color => :red, :background => :black).blink
+			puts "  #{@move[@wrong_count]}  '^^^' #{@spaces[@wrong_count]}  #{@p4}  ".colorize(:color => :red, :background => :black).blink
+			puts "===================================".colorize(:color => :red, :background => :black).blink
+			puts "                                   ".colorize(:color => :red, :background => :black).blink 
+			puts "Word: #{@game_word_array.join(" ")}   #{@space1}".colorize(:color => :red, :background => :black).blink 
+			puts "letters guessed: #{@guessed_letters.join(" ")} #{@space2}".colorize(:color => :red, :background => :black).blink 
+			puts "Word is #{@secret_word} #{@space3}".colorize(:color => :red, :background => :black).blink 
+			puts "Number of guesses left: #{5-@wrong_count}          ".colorize(:color => :red, :background => :black).blink 
+			puts "===================================".colorize(:color => :red, :background => :black).blink 
+			puts "=( GAME OVER                       ".colorize(:color => :red, :background => :black).blink 			
+			puts
+			exit
+		end
+
+	end
+
+	def game_start
+		puts "lets start the game!"
+		word_gen
+		until wrong_count == 5
+			game_over?
```

Comment:
Excellent use of a huh method here. `game_over?` is extremely expressive; telling me (the programmer reading your code), that you're making a decision or comparison to determine how to proceed.

The `game_over?` method itself is pretty large and has a couple of responsibilities. Ideally, huh methods should return `true` or `false`. So while this method is expressive in its intent here, I was surprised that it wasn't returning a boolean. I think, high level, a refactor could look something like:

```ruby
def game_start
  puts "let's start the game!"
  word_gen # a comment here about what word gen does
  until game_over? # return true/false
    guess # collects user guess and processes it
  end

  # everything after the `until` loop only happens once the look exits - so we know the game is over
  # the show_win and show_lose would be extracted from the if/else in your current `game_over?` method.
  if game_won? # now that it's over, did we win?
    show_win
  else
    show_lose
  end
```